Templates/Ros2FleetRobotTemplate/Template/Gem/Platform/Mac/PAL_mac.cmake

```diff
@@ -0,0 +1,13 @@
+# {BEGIN_LICENSE}
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+# {END_LICENSE}
+# This file is copied during engine registration. Edits to this file will be lost next
+# time a registration happens.
+
+set(PAL_TRAIT_${Name}_SUPPORTED TRUE)
```

Comment:
Not sure if we should claim support for Mac here. Or is the purpose of this file different?

Reply:
I would not treat support for MacOS as not-existing. I cannot see any particular reason for one to test it at Mac OS against installation from sources: https://docs.ros.org/en/humble/Installation/Alternatives/macOS-Development-Setup.html